datadog-checks-base/datadog_checks/checks/win/winpdh_base.py

```diff
@@ -60,13 +71,37 @@ def __init__(self, name, init_config, agentConfig, instances, counter_list):
                         self.log.error("Failed to make remote connection %s" % str(e))
                         return
 
+                datatypes = {}
+                precisions = instance.get('counter_data_types')
```

Comment:
for dev reference: could you add a comment on top of this detailing the expected format for `counter_data_types`?